tests/test_connector.py

```diff
@@ -43,8 +43,8 @@ def ssl_key():
 
 
 @pytest.fixture
-def unix_sockname(shorttmpdir):
-    sock_path = shorttmpdir / 'socket.sock'
+def unix_sockname(tmpdir):
```

Comment:
`tmpdir` -> `tmp_path`

Reply:
This was not done/changed.

Reply:
Yes, I understand.
A task for the next PR?

Reply:
=> https://github.com/aio-libs/aiohttp/pull/3555